docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md

```diff
@@ -291,6 +291,34 @@ controllers processing those resource that might impact system stability.
 
 </aside>
 
+### ExtensionConfig
+
+To register your runtime extension in the management cluster, apply an ExtensionConfig CRD in the desired namespace, 
```

Comment:
ExtensionConfig is not namespaced, thus it's not possible to deploy it into a namespace